app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -20,11 +20,12 @@ function require_ev_one() {
     exit 1
 }
 
-require_ev_one ETCD_IMG ETCD_TAG
+require_ev_one ETCD_IMG ETCD_IMG_TAG
```

Comment:
We haven't written any documentation or public announcements of etcd-wrapper. I dare to say it's OK to rename it at this point.